enhancements/builds/disable-jenkins-pipeline.md

```diff
@@ -0,0 +1,249 @@
+The Jenkins container image is also very large - it is in fact the largest image in the OpenShift payload today.
+Removing Jenkins from the OpenShift payload has been a longstanding goal that can reduce the time needed to pull the OCP payload at install/updgrade time.
+
+Red Hat does not have a strong connection to the Jenkins community, and has marginal influence when it comes to requested features and patches.
+Recently Red Hat tried to engage with a third party to develop an open source Kubernetes operator for Jenkins.
+That effort unfortunately fell apart, leading Red Hat to fork the operator in an attempt to continue its development.
+No other companies or individuals have contributed to the operator since, and its current form does not provide measurable value beyond what is currently available with the Jenkins templates.
+
+With the upcoming GA release of OpenShift Pipelines, we would like to decouple Jenkins from OpenShift.
+This would allow bug fixes in our Jenkins distribution to be released on its own cadence.
+Customers can continue to deploy Jenkins on OpenShift via its ImageStream and associated templates.
+In this fashion, Jenkins becomes just another application on OpenShift and does not retain any preferential status.
+
+### Goals
+
+* Disable the integration points between Jenkins and OpenShift.
+* Move Jenkins out of the OCP Payload
+* Document how JenkinsPipeline strategy builds can be migrated off of OpenShift.
+
+### Non-Goals
+
+* Migrate Jenkins users to OpenShift Pipelines/Tekton.
+* Stop distributing the Jenkins image and our associated OpenShift plugins.
+
+## Proposal
+
+### User Stories
+
+As a developer using Jenkins on OpenShift
+I want to continue using my Jenkinsfiles
+So that I can continue to use Jenkins for my CI/CD processes
+
+As an OpenShift release engineer
+I want to remove Jenkins from the OCP payload
+So that I can reduce the size of the payload
+And release Jenkins fixes on their own cadence
+
+### Implementation Details/Notes/Constraints [optional]
+
+Deprecation warnings for the Jenkins Pipeline Build Strategy will need to be updated in the next OpenShift release (OCP 4.N).
+Since Jenkins integrations have been officially been officially deprecated for well over one year, these warnings need to be strengthened so customers are fully aware that all direct Jenkins integrations will be removed.
+Like the use of other deprecated APIs, OpenShift must also alert users if a Jenkins pipeline build strategy is used in a Build or BuildConfig.
+
+In the OCP 4.(N+1) release, builds which use the Jenkins pipeline build strategy should fail immediately.
+All documentation related to Jenkins pipeline builds will need to be removed, and replaced with a statement that the Jenkins pipeline strategy has been disabled.
+The updated documenation should include instructions on how to move a Jenkinsfile off of OpenShift.
```

Comment:
the use of `curl` POST/Jenkins REST API and maybe the Jenkins CLI (though I think it may have fallen from favor as well) are both alternatives to comman line `oc start-build`

Reply:
either a mention here and/or in the migration/alternatives section works for me  ... where ever you deem fit @adambkaplan 

but the fact command line invocation of jenkins pipelines is available outside of `oc start-build` seems important to me